added cmd class where images can be passed one or many

Dennismwendwa opened the pull request.

kennithnichol commented on README.md:
Why have you deleted the README? can you please restore it?

kennithnichol commented on community-version.py:
Since the last few PRs have been merged this will likely conflict, can you please pull the latest and review to see if your changes still work?

Dennismwendwa replied:
updated the feature to work together with the other applied features. You can review it again now. Sorry for removing the README.md file. I deleted other files and deleted the file by mistake. Thanks

kennithnichol reviewed (changes requested):
Thank you, there are a few items to address, but overall this looks great.

Dennismwendwa commented:
@kennithnichol  updated the feature to work together with the other applied features. You can review it again now. Sorry for removing the README.md file. I deleted other files and deleted the file by mistake. Thanks

kennithnichol reviewed (approved):
Thank you for your updates!
